content/utilities/foundries-score/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Foundries Score
+date: 2021-10-03T12:37:08+03:00
+subtitle: Summary points of your unprospected foundries. Recalculates every 60 seconds by default.
```

Comment:
```suggestion
subtitle: Summary points of your unprospected foundries
```
Heh sorry shouldn't be here though, thats too long of a title for the front page